### PR TITLE
feat: support zh on home page

### PR DESCRIPTION
**Pull request by Keith-CY**
EN: https://nervos-official-website-git-i18n-zh-magickbase.vercel.app/
CN: https://nervos-official-website-git-i18n-zh-magickbase.vercel.app/zh

### CODE REVIEW

File: public/locales/en/mining.json
Comment (WhiteMinds) [inline public/locales/en/mining.json:2]: awesome 😆

**Review by WhiteMinds — CHANGES_REQUESTED** [review]

File: src/pages/ckbpage/index.page.tsx
Comment (Keith-CY) [reply src/pages/ckbpage/index.page.tsx:136]: I just found one page of `knowledge-base` and has imported `common` there by https://github.com/Magickbase/nervos-official-website/pull/108/commits/339da1042d89dd2c0c8e9d39be61160a5f224288

File: next-i18next.config.js
Comment (Keith-CY) [reply next-i18next.config.js]: Also blocked by the alert, updated by https://github.com/Magickbase/nervos-official-website/pull/108/commits/98b47e9e64e5c0459666612c4490d0f8e32ebdb8

File: src/components/Header/index.tsx
Comment (Keith-CY) [reply src/components/Header/index.tsx:1]: The `disabled` attribute on `Chinese` was removed by https://github.com/Magickbase/nervos-official-website/pull/108/commits/40b12220abcc8e5ccc9b91e559b469c834e1d56c

File: src/pages/ckbpage/index.page.tsx
Comment (Keith-CY) [reply src/pages/ckbpage/index.page.tsx]: Removed by https://github.com/Magickbase/nervos-official-website/pull/108/commits/a06f0719b86cc5a74992db45111a39d65a1c93aa

File: next-i18next.config.js
Comment (WhiteMinds) [reply next-i18next.config.js]: The following TODO and localeDetection can also be removed

File: src/pages/ckbpage/index.page.tsx
Comment (Keith-CY) [reply src/pages/ckbpage/index.page.tsx:136]: `common` was imported on `knowledge-base/[slug]` by https://github.com/Magickbase/nervos-official-website/pull/108/commits/0718517fc21025a9b4d203829e98f2d97266bb04

File: next-i18next.config.js
Comment (Keith-CY) [reply next-i18next.config.js]: Ah I recalled why `zh` is still disabled. The translation is done by gpt-3.5 and requires review from the foundation

Comment (Keith-CY) [reply next-i18next.config.js]: `zh` and `locale detection` were disabled again by https://github.com/Magickbase/nervos-official-website/pull/108/commits/82b28ac74b0e37a03e6bdf6e53abccc42d70037d because the content should be reviewed.

These updates could be merged first so community can work on the internationalization.

**Pull request merged**
